src/components/ui/PressCardList/Item/PressCardItem.module.css

```diff
@@ -0,0 +1,75 @@
+.root {
+  margin-bottom: 50px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 24px;
+  position: relative;
+}
+
+.header,
+.body {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.body {
+  gap: 30px;
+}
+
+.date {
+  font-weight: 500;
+  font-size: 12px;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  color: #aaa078;
+}
+
+.title {
+  font-weight: 600;
+  font-size: 20px;
+  letter-spacing: -0.01em;
+  text-transform: uppercase;
+  color: #f8f7f0;
+}
+
+.imgWrapper {
+  width: 100%;
+  height: 290px;

```

Comment:
@Yura33-dev To ensure correct image scaling, let's utilize the `aspect-ratio` property in the `.imgWrapper` component instead of setting the height directly. Remember to set `position: relative` for as our image is now has absolute. By allowing the image to fill its container, it maintains its correct aspect ratio automatically.

Please remove the following properties from the `.img` element to maintain cleaner code:
- `width: 100%;`
- `aspect-ratio: 3/2;`